QuestionAnswering/QA_ICL/data_generation/README.md

```diff
@@ -1,90 +1,155 @@
-# Data generation for QA applications using in-context learening
-
-## Introduction
-
-This directory contains scripts to generate three kinds of data: lexicon, simplified graph schema, and examples of data request generation.
+# Data Generation for QA_ICL
+
```

Comment:
This documentation contains a lot of relevant content but is either not properly structured or missing explanations for users. For instance, how could a user generate a new lexicon. 

I suspect that the code is tailored to Marie and isn't generalisable, so if that is true, mention this near the start, as a new user will be not be sure if they can generate new datasets if they run the script.